source/common/access_log/access_log_impl.cc

```diff
@@ -230,13 +230,13 @@ bool GrpcStatusFilter::evaluate(const StreamInfo::StreamInfo& info, const Http::
   //   3. Inferred from info HTTP status, if it exists.
   //
   // If none of those options exist, it will default to Grpc::Status::GrpcStatus::Unknown.
-  const std::array<absl::optional<Grpc::Status::GrpcStatus>, 3> optional_statuses = {
-      Grpc::Common::getGrpcStatus(response_trailers),
-      Grpc::Common::getGrpcStatus(response_headers),
-      info.responseCode() ? absl::optional<Grpc::Status::GrpcStatus>(
-                                Grpc::Utility::httpToGrpcStatus(info.responseCode().value()))
-                          : absl::nullopt,
-  };
+  const std::array<absl::optional<Grpc::Status::GrpcStatus>, 3> optional_statuses = {{
```

Comment:
I couldn't repro compile errors locally, but it looked like it was expecting braces around each individual object in the array. To do so I needed to add the double brace around the entire initializer list.

Let me know if there's a better way to do it.